tests/RocksDBEngine/ShaEventListener.cpp

```diff
@@ -21,18 +21,16 @@
 /// @author Lars Maier
 ////////////////////////////////////////////////////////////////////////////////
 
+#include <rocksdb/listener.h>
 #include "ApplicationFeatures/ApplicationServer.h"
-#include "Basics/Common.h"
-#include "Basics/FileUtils.h"
 #include "Basics/StringBuffer.h"
```

Comment:
Can we remove the StringBuffer include here now?

Reply:
StringBuffer is still use to append to the file name